packages/nomad/api.hcl

```diff
@@ -73,6 +73,9 @@ job "api" {
         REDIS_URL                     = "${redis_url}"
         # This is here just because it is required in some part of our code which is transitively imported
         TEMPLATE_BUCKET_NAME          = "skip"
+        ADMIN_TOKEN                   = var.admin_token
+        REDIS_URL                     = var.redis_url
+        DNS_PORT                      = 5353
```

Comment:
is there a particular reason to use a non-default DNS port?

Reply:
yes, there already running another DNS server on 53